### DISCUSSION TITLE
Full list of supported compressed files - analysis?

### DISCUSSION THREAD

**Discussion by RollingMoses** [Security-Onion-Solutions/securityonion#4384] [Unsupported Versions] [answered]
Is there a list somewhere that shows which compressed file types are supported for analysis through Security Onion? I found some items that said it can and, looking at Strelka docs specifically, it mentioned that it can, but I'm curious if any and all compressed file types are supported. For example, can it extract and analyze a .7z file? We're looking at deploying a specific compression tool and want to make sure it will be analyzed by our SO sensors.

**Comment by weslambert** [comment, marked as answer]
You can find the scanner list here: https://target.github.io/strelka/#/?id=scanner-list.  You can scan .7z as well. If you are already running Strelka on Security Onion, I'd recommend copying a single file to the `/nsm/strelka/unprocessed` directory in Security Onion and see what the results look like in `/nsm/strelka/strelka.log`.
